custom-remote-user-store-management/manager/pom.xml

```diff
@@ -0,0 +1,168 @@
+                        <goals>
+                            <goal>scr</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>${maven.bundle.plugins.version}</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>org.wso2.carbon.remote.user.store.manager.internal,</Private-Package>
+                        <Import-Package>
+                            <!--org.wso2.carbon.utils.*,-->
+                            <!--org.wso2.carbon.core.*,-->
+                            <!--org.osgi.framework,-->
+                            <!--org.osgi.service.component,-->
+                            <!--;resolution:=optional-->
+                        </Import-Package>
+                        <Export-Package>!org.wso2.carbon.remote.user.store.manager.internal,
+                            org.wso2.carbon.remote.user.store.manager.*
+                        </Export-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.2</version>
+                <configuration>
+                    <source>${wso2.maven.compiler.source}</source>
+                    <target>${wso2.maven.compiler.target}</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <wso2.maven.compiler.source>1.8</wso2.maven.compiler.source>
+        <wso2.maven.compiler.target>1.8</wso2.maven.compiler.target>
+        <httpcomponents-httpclient.wso2.version>4.3.1.wso2v2</httpcomponents-httpclient.wso2.version>
+        <maven-scr-plugin.version>1.22.0</maven-scr-plugin.version>
+        <httpclient.version>4.1.1</httpclient.version>
+        <org.wso2.carbon.utils.version>4.4.36</org.wso2.carbon.utils.version>
+        <carbon.kernel.version>4.4.35</carbon.kernel.version>
+        <json.wso2.version>3.0.0.wso2v1</json.wso2.version>
+        <carbon.identity.framework.version>5.12.257</carbon.identity.framework.version>
+        <commons-logging.version>1.2</commons-logging.version>
+        <org.apache.felix.scr.ds-annotations.version>1.2.4</org.apache.felix.scr.ds-annotations.version>
+        <maven.bundle.plugins.version>2.4.0</maven.bundle.plugins.version>
+    </properties>
+</project>
```

Comment:
Add a new line.

Reply:
Fixed in 51041691a23b06016ed6fa558504cc92b89c40b3 and 8a4b4fe793b231e0bd48d328c076d4cfd9494c5e